content/learning-paths/laptops-and-desktops/mca-godbolt/running_mca.md

```diff
@@ -0,0 +1,350 @@
+---
+title: Running MCA with Arm assembly
+weight: 2
+### FIXED, DO NOT MODIFY
+layout: learningpathall
+---
+
+### What is MCA?
+MCA stands for Machine Code Analyser. It is a performance analysis tool that uses information available in [LLVM](https://github.com/llvm/llvm-project) to measure performance on a specific CPU.
+
+
```

Comment:
I think it would make sense to have a lightweight description of how instruction scheduling works as well as a brief definition of some of the terms that are used later on, e.g. resource pressure, ... The learning path will target people who are not expert in the field of instruction scheduling and performance analysis and I fear they are going to be lost.

Reply:
I see what you mean. I can add another section to describe those terms.

Reply:
I've added a new section defining the terms you mentioned and some I thought might be worth explaining. I found there was no need to refactor sections "What is MCA?" and "How can MCA be useful?". If you think otherwise let me know and I will make edits accordingly.